Library/Homebrew/utils/shell_env.sh

```diff
@@ -0,0 +1,71 @@
+keepkeys=(
+HOMEBREW_*
+GEM_*
+HOME
+PWD
+LANG
+PATH
+)
+
+prune-shell-env() {
+  # Associative arrays are only a Bash 4 feature
+  #declare -A aEnv
+
+  local verbose
+  
+  local envPATH=`which env`
```

Comment:
favour `env_path` or similar for local variable names and `$(...)` although in this case you can just use `env` rather than looking up the path.